extensions/wikia/AdEngine/js/slot/premiumFloatingMedrec.js

```diff
@@ -0,0 +1,17 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.premiumFloatingMedrec', [
+	'ext.wikia.adEngine.adContext',
+], function (
+	adContext
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "adContext". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)